packages/devextreme/js/ui/tabs.js

```diff
@@ -519,6 +536,32 @@ const Tabs = CollectionWidget.inherit({
         this._toggleTabsHorizontalClass(!isVertical);
     },
 
+    _getTabIconPositionClass() {
+        const position = this.option('iconPosition');
+
+        switch(position) {
+            case ICON_POSITION.top:
+                return TABS_ICON_POSITION_CLASS.top;
+            case ICON_POSITION.end:
+                return TABS_ICON_POSITION_CLASS.end;
+            case ICON_POSITION.bottom:
+                return TABS_ICON_POSITION_CLASS.bottom;
+            case ICON_POSITION.start:
+            default:
+                return TABS_ICON_POSITION_CLASS.start;
+        }
+    },
+
+    _toggleIconPositionClass() {
+        for(const key in TABS_ICON_POSITION_CLASS) {
```

Comment:
in lint rules for TS for in cycle is prohibited. may be we will not add for in to new code?

Reply:
Let's discuss reasons.